administrator/components/com_joomgallery/models/fields/fineuploader.php

```diff
@@ -0,0 +1,382 @@
+      },
+      fileTemplate: 'qq-template',
+      classes: {
+          success: 'alert-success',
+          fail: 'alert-error',
+          debugText: 'qq-upload-debug-text',
+          thumb: 'qq-upload-thumb',
+          options: 'qq-upload-options',
+          note: 'qq-upload-note'
+      },
+      validation: {
+        allowedExtensions: ['jpg', 'jpeg', 'jpe', 'gif', 'png'],
+        acceptFiles: 'image/*',
+        sizeLimit: <?php echo $fileSizeLimit; ?>
+      },
+      messages: {
+        typeError: '{file}: ' + '<?php echo JText::_('COM_JOOMGALLERY_COMMON_ALERT_WRONG_EXTENSION', true); ?>',
+        sizeError: '{file}: ' + '<?php echo JText::sprintf('COM_JOOMGALLERY_UPLOAD_OUTPUT_MAX_ALLOWED_FILESIZE', $fileSizeLimit, array('jsSafe' => true)) ?>',
+        fileNameError: '{file}: ' + '<?php echo JText::_('COM_JOOMGALLERY_COMMON_ALERT_WRONG_FILENAME', true); ?>',
+        fileNameDouble: '{file}: ' + '<?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_ALERT_FILENAME_DOUBLE', true); ?>',
+        minSizeError: '{file}: ' + '<?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_ALERT_FILE_TOO_SMALL', true); ?>' + ' {minSizeLimit}.',
+        emptyError: '{file} : '  + '<?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_ALERT_FILE_EMPTY', true); ?>',
+        noFilesError: '<?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_ALERT_NO_FILES', true); ?>',
+        onLeave: '<?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_ALERT_ON_LEAVE', true); ?>'
+      },
+      debug: true,
+      maxConnections: 1,
+      disableCancelForFormUploads: true,
+      callbacks: {
+        onComplete: function(id, fileName, response) {
+          var item = this.getItemByFileId(id);
+<?php if($isMini):
+        if($insertOptions): ?>
+          // Display options if the upload was successful and if the image is published
+          if(response.success && !document.id('published0').checked) {
+            displayInsertOptions(this, item, fileName, response);
+          }
+<?php   else: ?>
+          // Reload the page if published images were uploaded (so the user can see them instantly on the page)
+          if(response.success && !document.id('published0').checked) {
+            window.parent.SqueezeBox.addEvent('onClose', function(){window.parent.location.reload();});
+          }
+<?php   endif;
+      endif; ?>
+          if(response.debug_output) {
+            var element = item.getElementsByClassName("qq-upload-debug-text-selector")[0];
+            element.innerHTML = response.debug_output;
+          }
+          if(this.requestParams.hasOwnProperty("filecounter")) {
+            this.requestParams.filecounter =  this.requestParams.filecounter + 1;
+            this.setParams(this.requestParams);
+          }
+          if(jQuery('#<?php echo $prefix; ?>generictitle').length > 0) {
+            if(!jQuery('#<?php echo $prefix; ?>generictitle').prop('checked')) {
+              jQuery('#<?php echo $prefix; ?>imgtitleid-' + id).remove();
```

Comment:
Before it was different for backend and frontend. The image title input field was just made readonly. I found this useful, even if you can no longer change the title. This is just a matter of taste.